Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs

```diff
@@ -161,7 +161,7 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateTextDecorations();
```

Comment:
```suggestion
				UpdateTextDecorations(false);
```

Reply:
Am I missing something here? True should be used if it's a new element, false if it's an updated property on an existing one.

Reply:
Nothing was specified so it was causing a compile error. 

This UpdateTextDecorations is part of the ElementPropertyChanged right?

https://github.com/xamarin/Xamarin.Forms/blob/221f43d689393bc58811d602c93fc7a74f348a5c/Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs#L161

Reply:
Oh oops, that was added in a PR separate from the one I did this feature in so I missed it. I could have sworn I ran this on all platforms, that's embarrassing.

Reply:
Ah, I think I tested this before that change went in and then rebased with out building again before I filed this.